src/java/org/apache/cassandra/service/reads/ReadCallback.java

```diff
@@ -105,6 +160,25 @@ public void awaitResults() throws ReadFailureException, ReadTimeoutException
          */
         int received = resolver.responses.size();
         boolean failed = failures > 0 && (blockFor > received || !resolver.isDataPresent());
+        WarningCounter warnings = warningCounter;
+        if (warnings != null)
+        {
+            if (warnings.tombstoneAborts.get() > 0)
+            {
+                String msg = tombstoneAbortMessage(warnings.tombstoneAborts.get(), warnings.maxTombstoneAbortsCount.get(), command.toCQLString());
+                ClientWarn.instance.warn(msg + " with " + command.loggableTokens());
+                logger.warn(msg);
+                cfs().metric.clientTombstoneAborts.mark();
```

Comment:
should we inc if the query doesn't fail?  kinda think we should inc at the throws?